src/doc/style-guide/src/expressions.md

```diff
@@ -341,12 +341,6 @@ impl SomeType {
             .extra_info =
                 long_long_long_long_long_long_long_long_long_long_long_long_long_long_long;
 
-        self.array[array_index as usize]
-            .as_mut()
-            .expect("thing must exist")
-            .extra_info
-                + long_long_long_long_long_long_long_long_long_long_long_long_long_long_long;
-
```

Comment:
It _might_ be worth leaving this but with line 348 deindented. we don't need it for rustfmt and implementation purposes, but it would be very explicit that at least for now the expectation would be:


```rust
        self.array[array_index as usize]
            .as_mut()
            .expect("thing must exist")
            .extra_info
            + long_long_long_long_long_long_long_long_long_long_long_long_long_long_long;
````

Reply:
@calebcartwright I can understand the idea of including it to show how assignments are different from other binops, but I feel like including it *here* would be confusing the example for assignments. I think if we're going to include examples of other formatting, those examples should be elsewhere in the guide.